modules/order/config/install/views.view.line_item_summary.yml

```diff
@@ -0,0 +1,397 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - commerce_order
+    - commerce_price
+id: line_item_summary
+label: 'Line item summary'
+module: views
+description: ''
+tag: ''
+base_table: commerce_line_item
```

Comment:
Doesn't line_item_table make more sense, cause it mirrors the formatter name (and 1.x)?

I was initially surprised that this is not an order view, but I see 1.x did the same. I guess that means the order total is not desired in the footer?